cmd/devenv/provision/docker_for_mac.go

```diff
@@ -17,17 +17,20 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// startDockerForMac starts Docker for Mac and waits for it to be ready.
 func startDockerForMac(ctx context.Context, d dockerclient.APIClient, log logrus.FieldLogger) error {
 	// Give Docker for Mac time to stop.
 	time.Sleep(2 * time.Second)
 
-	cmd := exec.CommandContext(ctx, "open", "-a", "Docker")
-	if out, err := cmd.CombinedOutput(); err != nil {
-		return errors.Wrapf(errors.Wrap(err, string(out)), "failed to open Docker for Mac (try starting it manually)")
-	}
-
-	ticker := time.NewTicker(7 * time.Second)
+	ticker := time.NewTicker(5 * time.Second)
 	for {
+		// Always try to start Docker for Mac, even if it's already running.
+		// This doesn't appear to have any side effects.
+		cmd := exec.CommandContext(ctx, "open", "-a", "Docker")
+		if out, err := cmd.CombinedOutput(); err != nil {
+			return errors.Wrapf(errors.Wrap(err, string(out)), "failed to open Docker for Mac (try starting it manually)")
```

Comment:
Nit: unsure why this is `errors.Wrapf` and not just `errors.Wrap`.
```suggestion
			return errors.Wrap(errors.Wrap(err, string(out)), "failed to open Docker for Mac (try starting it manually)")
```